swan-lake/integration-tools/graphql-tool.md

```diff
@@ -8,15 +8,57 @@ active: graphql-tool
 intro: The Ballerina GraphQL tool makes it easy to start the development of a GraphQL client. It generates the client skeletons in Ballerina for a given GraphQL schema and a GraphQL document. In addition, the GraphQL tool supports generating the GraphQL schema for a given Ballerina GraphQL service and writing it to a file in GraphQL schema definition language (SDL).
 --- 
 
+## Usage
+
 The Ballerina GraphQL tooling support provides the following capabilities.
 
-1. Generating a Ballerina client from a given GraphQL config file configured with a GraphQL schema, which is specified by the GraphQL schema definition language and a GraphQL document/documents.
+1. Generate a Ballerina client from a given GraphQL config file configured with a GraphQL schema, which is specified by the GraphQL schema definition language and a GraphQL document/documents.
 
-2. Generating multiple Ballerina modules from a given GraphQL config file configured with multiple GraphQL projects.
+2. Generate multiple Ballerina modules from a given GraphQL config file configured with multiple GraphQL projects.
 
 3. Generate the GraphQL schema for a given Ballerina GraphQL service(s) and write the schema(s) to a file/files using the GraphQL schema definition language.
 
-> **Prerequisites:** Install the latest <a href="https://ballerina.io/downloads/" target="_blank">Ballerina Swan Lake distribution</a>.
+## Available parameters
+
+The below command-line arguments can be used with the command.
+
+### GraphQL to Ballerina 
```

Comment:
As mentioned above, there are two main functionalities when it comes to `GraphQL to Ballerina`. 
- Client generation
- Service generation

Need to include both here.

Reply:
These changes are to improve the ecxisting content and will go live before Update 8. Hence, the content for Update 8 will be added later when syncing the Up8 branch to the master branch.

Reply:
+1, as long as those changes won't go missing.